CMakeLists.txt

```diff
@@ -157,29 +146,24 @@ if (BUILD_WERROR)
 endif()
 
 # Platform-specific compiler switches
-if(${CMAKE_C_COMPILER_ID} STREQUAL "Clang")
-    add_compile_options(-Werror)
-endif()
-if(${CMAKE_C_COMPILER_ID} MATCHES "(GNU|Clang)")
-    add_compile_options(-Werror)
```

Comment:
BUILD_WERROR handles this.

Reply:
Also CMAKE_C_COMPILER_ID doesn't exist anymore. Now it's just CMAKE_CXX_COMPILER_ID